packages/daemon/src/context/ContextMutations.js

```diff
@@ -111,11 +111,7 @@ export default class ContextMutations {
     })
   }
 
-  async createContactFromPeer(
-    userID: ID,
-    peerID: ID,
-    aliasName?: string,
-  ): Promise<Contact> {
+  createContactFromPeer(userID: ID, peerID: ID, aliasName?: string): Contact {
```

Comment:
Shouldn't the vault get saved after creating this contact?

Reply:
This is handled by the `autoSaveVault` subscription